Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -147,11 +147,8 @@ protected override async void Start()
                 TeleportSystem.Register(gameObject);
             }
 
-            if (InputSystem == null)
```

Comment:
Not sure about removing this null check, seems quite valid

Reply:
We don't need it bc we wait until the input system is valid in the line below on 

`await WaitUntilInputSystemValid;`

Reply:
Unless the InputSystem is off and I add the BaseControllerPointer to an object manually (or other component consuming it).  Which means it will sit forever in this loop.

Reply:
> Which means it will sit forever in this loop.

It's not a loop, as it's just a continuation. No overhead there.

Besides why would you disable the input system then put something that depends on the input system directly in the scene yourself unless you're asking for trouble.